pkg/client/testsuites/v3/testsuites.go

```diff
@@ -25,7 +25,7 @@ const (
 var testsuiteSecretDefaultLabels = map[string]string{
 	"testkube":           testkubeTestsuiteSecretLabel,
 	"testkubeSecretType": "variables",
-	"visibility":         "testkube",
+	"createdBy":          "testkube",
```

Comment:
Do we need backwards compatibility here?

Reply:
doesn't looks so, we didn't have UI for this feature before and even we didn't use it